docs/gettingstarted/autoupdate.md

```diff
@@ -40,3 +40,9 @@ This is fairly self-explanatory, but some notes:
 * Manually inputting data can be chore. You can configure your `build.gradle` to automatically update this file when building a release, as Groovy has native JSON parsing support. Doing this is left as an exercise to the reader.
 
 Two concrete examples can be seen here for [Charset](https://gist.githubusercontent.com/Meow-J/fe740e287c2881d3bf2341a62a7ce770/raw/bf829cdefc84344d86d1922e2667778112b845b1/update.json) and [Botania Unofficial](https://gist.githubusercontent.com/Meow-J/1299068c775c2b174632534a18b65fb8/raw/42c578cf2303aa76d8900f5fdc6366122549d2a8/update.json).
+
+Retrieving Update Check Results
+-------------------------------
+
+You can retrieve the results of the Forge Update Checker using `ForgeVersion.getResult(ModContainer)`. The returned object has a field `status` which indicates the status of the version check. Example values: `FAILED` (the version checker couldn't connect to the URL provided), `UP_TO_DATE`, `OUTDATED`, `BETA_OUTDATED`. The status will be `PENDING` if the result requested has not finished yet; in that case, you should try again in a little bit. Otherwise, the returned object will also have the target version and any changelog lines, as specified in `update.json`.
+You can obtain your own `ModContainer` to pass to this method using `Loader.instance().activeModContainer()`, or any other mod's `ModContainer` using `Loader.instance().getIndexedModList().get(<modid>)`.
```

Comment:
^ any response to that comment? You want the modcontainer not the actual mod object.